packages/tradershub/src/features/cfd/flows/CTrader/AvailableCTraderAccountsList/AvailableCTraderAccountsList.tsx

```diff
@@ -1,5 +1,6 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { useActiveTradingAccount, useCreateOtherCFDAccount } from '@deriv/api';
+import { Provider } from '@deriv/library';
```

Comment:
Question:
Now we have `@deriv/ui` in the works does this mean `@deriv/library` is now irrelevant?

Reply:
Provider does not exist in ui library yet. Once that is implemented will replace it